tests/test-cases/Group23-VIC-Machine-Service/23-03-VCH-Create.robot

```diff
@@ -70,12 +70,14 @@ Get VCH ${name}
 Pull Busy Box
     [Arguments]    ${docker_params}    ${expected_rc}    ${expected_output}
 
+    # Because this is used to verify the whitelist registry functionality, we can't use ${busybox}!
     ${rc}  ${output}=  Run And Return Rc And Output  docker -H ${docker_params} pull busybox
 
     Should Be Equal As Integers    ${rc}    ${expected_rc}
     Should Contain    ${output}    ${expected_output}
 
 
+# This keyword tries to get the docker host parameters in two different ways to help identify the cause of failure, should it occur.
 Get Docker Host Params
     [Arguments]    ${vch_name}
     Run Keyword And Continue On Failure    Wait Until Keyword Succeeds    30x    10s    Get Docker Params API    ${vch_name}
```

Comment:
I would still prefer `Wait For VCH Initialization` present before calling the `Get Docker Params API` so that we're assured that we get back the live data and not a cached set of IP addresses because a VCH has just been restarted.

The other issue I have with this is we don't report a failure if it occurs. At a minimum a message such as:
```
Log  Failed to get docker connection parameters from service API  level=WARN
```

Reply:
As discussed in person, `Run Keyword And Continue On Failure` without retries will ensure failures are always reported.